.github/workflows/release.yaml

```diff
@@ -5,9 +5,14 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+
 jobs:
   release:
     # Don't run on forks. We can and should only release from the main repo.
+    permissions:
+      contents: write  # for Git to git push
```

Comment:
This workflow also creates a PR like this one https://github.com/lit/lit/pull/4399 via `changesets/action`, would it require `pull_requests: write` ?

Reply:
Yeah, it would 
I'll update it.

Reply:
Fix it. Thanks!